react/features/always-on-top/AlwaysOnTop.js

```diff
@@ -169,15 +175,13 @@ export default class AlwaysOnTop extends Component<*, State> {
 
         return (
             <div id = 'videoNotAvailableScreen'>
-                {
-                    avatarURL
-                        ? <div id = 'avatarContainer'>
-                            <img
-                                id = 'avatar'
-                                src = { avatarURL } />
-                        </div>
-                        : null
-                }
+                <div id = 'avatarContainer'>
+                    <StatelessAvatar
+                        defaultAvatar = { this._defaultAvatar }
+                        id = 'avatar'
+                        initials = { getInitials(displayName) }
```

Comment:
There is a difference between displayName here and web. In AoT, the display name is actually the formatted display name, which could be the default "Fellow Jitster" whereas web could use an empty string for a name.